File: chapters/sorting_searching/bubble/code/js/bubble.js
function bubbleSort(arr) {
  for (let r = arr.length -1; r > 0; r--) {
    for (let i = 0; i < r; i++) {
      if (arr[i] > arr[i + 1]) {
        let tmp = arr[i];
        arr[i] = arr[i + 1];
        arr[i + 1] = tmp;
      }
    }
  }
}

File: chapters/sorting_searching/bubble/code/java/bubble.java
static void bubbleSort(int[] arr) {
    for (int r = arr.length - 1; r > 0; r--) {
        for (int i = 0; i < r; i++) {
            if(arr[i] > arr[i + 1]) {
                int tmp = arr[i];
                arr[i] = arr[i + 1];
                arr[i + 1] = tmp;
            }
        }
    }
}
